chromedriver_autoinstaller/utils.py

```diff
@@ -219,12 +214,19 @@ def get_matched_chromedriver_version(chrome_version, no_ssl=False):
     """
     # Newer versions of chrome use the CfT publishing system
     if chrome_version >= "115":
-        version_url = "googlechromelabs.github.io/chrome-for-testing/known-good-versions.json"
-        version_url = "http://" + version_url if no_ssl else "https://" + version_url
-        good_version_list = json.load(urllib.request.urlopen(version_url))
-        for good_version in good_version_list["versions"]:
-            if good_version["version"] == chrome_version:
-                return chrome_version
+        browser_major_version = get_major_version(chrome_version)
+        #version_url = "googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json"
+        #version_url = "googlechromelabs.github.io/chrome-for-testing/last-known-good-versions-with-downloads.json"
+        version_url = "googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json"
+        version_url = f"http://{version_url}" if no_ssl else f"https://{version_url}"
+        latest_version_per_milestone_list = json.load(urllib.request.urlopen(version_url))
+        # drill through object to make sure each key exists, to determin in driver download is available
+        if browser_major_version in latest_version_per_milestone_list['milestones']:
+            milestone = latest_version_per_milestone_list['milestones'][browser_major_version]
+            if 'downloads' in milestone:
+                if 'chromedriver' in milestone['downloads']:
+                    download_options = milestone['downloads']['chromedriver']
+                return browser_major_version, download_options
```

Comment:
I missed this indent in the first commit in #57 
```suggestion
                    download_options = milestone['downloads']['chromedriver']
                    return browser_major_version, download_options
```